src/Data/Nat/Show.agda

```diff
@@ -8,29 +8,104 @@
 
 module Data.Nat.Show where
 
-open import Data.Nat
+open import Data.Char as Char using (Char)
+open import Data.Digit using (showDigit; toDigits)
+open import Function   using (_∘_; _$_)
+open import Data.List  using (List; []; _∷_; map; reverse)
+open import Data.Nat   using (ℕ; _≟_; suc; pred; _+_; _*_; _<_; _>_; z≤n; s≤s;
+                                                                        _≤?_)
+open import Data.Nat.DivMod     using (_div_; _%_; a≡a%n+[a/n]*n)
+open import Data.Product        using (proj₁)
+open import Data.Nat.Properties using
+                     (<-irrefl; m≢0⇒suc[pred[m]]≡m; +-comm; *-identityʳ;
+                                         m≤m+n; *-monoʳ-<; module ≤-Reasoning)
+open import Data.String as Str using (String)
```

Comment:
I think it's better to import it as `String` rather than `Str`. Not that long and much more meaningful to read.

Reply:
Done.